Include/pystate.h

```diff
@@ -101,9 +115,7 @@ typedef struct _ts {
     PyObject *curexc_value;
     PyObject *curexc_traceback;
 
-    PyObject *exc_type;
-    PyObject *exc_value;
-    PyObject *exc_traceback;
+    PyExcState exc_state;
```

Comment:
PyExcState structs form a stack (hence the "previous" field), but there is only one current exception.